packages/clerk-js/src/ui/common/authForms/Header.tsx

```diff
@@ -37,6 +40,7 @@ export function Header({
         />
       )}
       {error && <Error style={errorStyle}>{error}</Error>}
+      {info && <Info>{info}</Info>}
```

Comment:
🔧 Let's introduce an Alert component with two variants/flavors for error and info. It's a more scalable approach as we will add more use cases to our components.

Reply:
Introduced `Alert` component.

Do we need to keep the `Error` component around for backwards compatibility?